common/src/main/java/com/sequenceiq/cloudbreak/event/ResourceEvent.java

```diff
@@ -190,7 +190,7 @@
     STACK_REMOVING_INSTANCE_FINISHED("stack.removing.instance.finished"),
     STACK_REMOVING_INSTANCE_FAILED("stack.removing.instance.failed"),
     STACK_INFRASTRUCTURE_BOOTSTRAP("stack.infrastructure.bootstrap"),
-    STACK_GATEWAY_CERTIFICATE_CREATE_FAILED("stack.gateway.certificate.create.failed"),
+    STACK_GATEWAY_CERTIFICATE_CREATE_SKIPPED("stack.gateway.certificate.create.skipped"),
```

Comment:
Since it's an enum I would rather go with an extension and remove it after the next release.